src/main/java/de/dhbw/woped/process2text/controller/P2TController.java

```diff
@@ -32,7 +31,6 @@ public class P2TController {
    * @param body The process model in plain text format.
    * @return The translated text.
    */
-  @ApiOperation(value = "Translate a process model into human readable text.")
```

Comment:
Why did we remove this? 
It'll be used in the Swagger UI to explain what the API endpoint does.